io.github.beamerpresenter.beamerpresenter.yml

```diff
@@ -0,0 +1,65 @@
+app-id: io.github.beamerpresenter.beamerpresenter
+branch: master
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: beamerpresenter
+build-options:
+  strip: true
+  no-debuginfo: true
+finish-args:
+  - "--socket=x11"
+  - "--socket=wayland"
+  - "--socket=pulseaudio"
+  - "--filesystem=xdg-documents"
+  - "--filesystem=xdg-download"
+  - "--filesystem=xdg-public-share"
+  - "--filesystem=xdg-desktop"
+  - "--filesystem=xdg-config/beamerpresenter"
+  - "--own-name=io.github.beamerpresenter.beamerpresenter"
+modules:
+  - name: poppler
+    buildsystem: cmake-ninja
+    config-opts:
+    - "-DCMAKE_INSTALL_PREFIX=/app"
+    - "-DCMAKE_INSTALL_LIBDIR=/app/lib"
+    - "-DCMAKE_INSTALL_INCLUDEDIR=/app/include"
+    - "-DENABLE_LIBOPENJPEG=none"
+    - "-DENABLE_BOOST=OFF"
+    - "-DENABLE_UTILS=OFF"
+    - "-DENABLE_LIBCURL=OFF"
+    - "-DENABLE_QT5=ON"
+    - "-DENABLE_QT6=OFF"
+    - "-DENABLE_GLIB=OFF"
+    - "-DENABLE_GOBJECT_INTROSPECTION=OFF"
+    - "-DENABLE_GTK_DOC=OFF"
+    - "-DENABLE_CPP=OFF"
+    cleanup:
+    - "/bin"
+    sources:
+    - type: archive
+      url: https://poppler.freedesktop.org/poppler-22.09.0.tar.xz
+      sha256: d7a8f748211359cadb774ba3e18ecda6464b34027045c0648eb30d5852a41e2e
+  - name: beamerpresenter
+    buildsystem: cmake-ninja
+    config-opts:
+    - "-DQT_VERSION_MAJOR=5"
+    - "-DQT_VERSION_MINOR=15"
+    - "-DUSE_POPPLER=ON"
+    - "-DUSE_MUPDF=OFF"
+    - "-DUSE_QTPDF=OFF"
+    - "-DUSE_EXTERNAL_RENDERER=OFF"
+    - "-DUSE_TRANSLATIONS=ON"
+    - "-DGIT_VERSION=OFF"
+    - "-DAPPID_NAME=io.github.beamerpresenter.beamerpresenter"
+    - "-DINSTALL_METAINFO=ON"
+    - "-DCMAKE_BUILD_TYPE=Release"
+    - "-DCMAKE_INSTALL_PREFIX=/app"
+    - "-DCMAKE_INSTALL_SYSCONFDIR=/app/etc"
+    - "-DCMAKE_INSTALL_DATAROOTDIR=/app/share"
+    - "-DCMAKE_INSTALL_LIBDIR=/app/usr/lib"
+    - "-DCMAKE_INSTALL_INCLUDEDIR=/app/usr/include"
```

Comment:
never do `/app/usr`. it's not fsb compliant and trigger some other bugs with appstream.

You shouldn't need to even have to set these anyway.